Update newsletter sponsorship pricing (#2162)
writethedocs/www: docs/include/conf/sponsorship/demographics.rst

> The most common job title is **Technical Writer**,
> but we have many other people who attend our conferences:
> 

Reviewer: ```suggestion
The most common job title is **Technical Writer**,
but people with many other roles also attend our conferences:

```

Driveby not in the PR, soz. 
Also, shouldn't we delete this and use `docs/include/demographics.rst` ?
